src/pages/molecules/itemCartBtn/ItemCartBtns.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
```

Comment:
Why are these not having index file?

Reply:
Do we need to create index file for helper functions also? or We need to create only for components?